tests/break-calls/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -116,3 +116,25 @@ runtimeAgent.getProperties(
 
 ================================================================================
 `;
+
+exports[`react.js 1`] = `
```

Comment:
We should break it if it's too long.

**Prettier pr-5608**
[Playground link](https://deploy-preview-5608--prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AEAFnADYkQDqEATiQCYAUAlPsADpRszoDOcAoqlRxsDZvgC8APlYcuXAPQL8dCPh4QAtnBiEcUAOZyYAXwA0+ANoAHahGs8AdADcAhiXRwLt+07cevfB8HF3dPbzsQ-3CgyL8wwOD4gIjfUJTYtOjEuPSYpLycrISLAF0mDhMOEDMQe1xoHmRQV2o7AHcABVaEJpRXZwgcOhqQACNqVzAAax0AZWsp-QNkGGpwkEIYTRIKPXgeRbA4Od6cXGdzgE9kcB4m2v0+ahhOyYNNV2RUdz5agCseAAPABCkxm81c2gAMvo4N9fl4QICgXNliQ4ABFdAQeAIkh-ECLajPW5jVxjK7kKCjWz6GAUYa6ZAADgADLUfHwKJNrLdbHBns54bVqHAAI7oHBit6uD5fJA-AlIviaHCrdYq9FYnF4xWI2owCmMujMpAAJkNkxwJGWAGEtJ9blBoCKQLw4AAVCl9JV8EwmIA)
```sh
--parser babylon
```

**Input:**
```jsx
function helloWorld() {
	useEffect(() => {
		// do something
	}, [props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, ])
}

```

**Output:**
```jsx
function helloWorld() {
  useEffect(() => {
    // do something
  }, [props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value]);
}

```

Reply:
I tried to debug this a bit, but I couldn’t figure out how to get this to work. Somehow, the breaking only occurs if the call isn’t grouped: 

**Prettier 1.15.3**
[Playground link](https://prettier.netlify.com/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AEAFnADYkQDqEATiQCYAUAlPsADpRszoDOcAoqlRxsDZvgC8APlYcuXAPQL8dCPh4QAtnBiEcUAOZyYAXwA0+ANoAHahGs8AdADcAhiXRwLt+07cevfB8HF3dPbzsQ-3CgyL8wwOD4gIjfUJTYtOjEuPSYpLycrISLAF0mDhMOEDMQe1xoHmRQV2o7AHcABVaEJpRXZwgcOhqQACNqVzAAax0AZWsp-QNkGGpwkEIYTRIKPXgeRbA4Od6cXGdzgE9kcB4m2v0+ahhOyYNNV2RUdz5agCseAAPABCkxm81c2gAMvo4N9fl4QICgXNliQ4ABFdAQeAIkh-ECLajPW5jVxjK7kKCjWz6GAUYa6ZAADgADLUfHwKJNrLdbHBns54bVqHAAI7oHBit6uD5fJA-AlIviaHCrdYq9FYnF4xWI2owCmMujMpAAJkNkxwJGWAGEtJ9blBoCKQLw4AAVCl9JV8EwmIA)
```sh
--parser babylon
```

**Input:**
```jsx
function helloWorld() {
	useEffect(() => {
		// do something
	}, [props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, props.value, ])
}

```

**Output:**
```jsx
function helloWorld() {
  useEffect(
    () => {
      // do something
    },
    [
      props.value,
      props.value,
      props.value,
      props.value,
      props.value,
      props.value,
      props.value,
      props.value,
      props.value,
      props.value,
      props.value
    ]
  );
}

```

The best solution I could figure out for this is to special-case printing `call(() => { arrow with block }, [array])`